### ISSUE TITLE
Golf7 GTD 

### ISSUE THREAD

**Issue by Pesroozza** [jilleb/mqb-soundaktor#45]
hello, is it possible to change the sw in golf7GTD to make it louder? 
Sw:5G0907160

**Comment by zzHAMZAzz** [comment]
Of course, I'd use one of the few FL_5G0907159___0014 firmwares, they are altered to use the GTI sound / GTI Clubsport which seem louder than the GTD one. Flash each one and see which you find best.

[FL_5G0907159___0014_UNLOCKED_1.odx](https://raw.githubusercontent.com/jilleb/mqb-soundaktor/main/Firmwares/FL_5G0907159___0014_UNLOCKED_1.odx)
[FL_5G0907159___0014_UNLOCKED_2.odx](https://raw.githubusercontent.com/jilleb/mqb-soundaktor/main/Firmwares/FL_5G0907159___0014_UNLOCKED_2.odx)
[FL_5G0907159___0014_UNLOCKED_3.odx](https://raw.githubusercontent.com/jilleb/mqb-soundaktor/main/Firmwares/FL_5G0907159___0014_UNLOCKED_3.odx)

**Comment by Lapiz8Blue** [comment]
I thought that the 159 firmware shouldn't have to be flashed on 160 modules.

Is this now no problem?
